Add a link to add monitor when the monitor status is inactive

yashwin opened the pull request:
#### Proposed Changes

* This PR adds a link to activate the monitor to the inactive monitor state.

#### Testing Instructions

**Prerequisites**

Since this change is made specifically for agencies, you will have to set yourself(partner) as an agency - 2c49b-pb. Make sure to switch it back to the previous type.

**Instructions**

1. Run `git checkout update/add-monitor-inactive-state-link` and `yarn start-jetpack-cloud`
2. Visit http://jetpack.cloud.localhost:3000/, and you'll be redirected to http://jetpack.cloud.localhost:3000/dashboard.
3. Click on `-` on the monitor column, and verify that you are being redirected to https://<site>/wp-admin/admin.php?page=jetpack#/settings in a new tab.

`Note` - Add a new site if the monitor is already active for your sites.

<img width="820" alt="Screenshot 2022-06-06 at 10 52 34 PM" src="https://user-images.githubusercontent.com/10586875/172212648-22189b1e-ca6d-4e2b-accd-a6b65deaabee.png">


Related to 1202076982646589-as-1202396957069327

yashwin committed:
Add monitor add link when monitor status is inactive

rcanepa commented:
The link appears and works correctly, but I'm seeing an error in the console every time I click the CTA. The error is this:
```
tracks.ts:215 Uncaught TypeError: Cannot read properties of undefined (reading 'startsWith')
    at recordTracksEvent (tracks.ts:215:18)
    at recordTracksEvent (tracks.js:18:83)
    at Object.tracks (middleware.js:30:88)
    at eval (middleware.js:76:40)
    at Array.forEach (<anonymous>)
    at dispatcher (middleware.js:65:17)
    at eval (middleware.js:99:9)
    at eval (index.js:17:83)
    at eval (middleware-calypso.js:221:10)
    at eval (middleware.js:86:12)
```

yashwin commented:
@rcanepa I have fixed the issue reported by you above, thanks for the same. I have also made some changes. We will still have `-` instead of `+Add` with the link opening in a new tab according to the suggestion by the design team. I have also updated the PR description.

rcanepa reviewed (approved):
LGTM now.

Thanks for fixing the issue.
